### ISSUE TITLE
company-box-frontend error Wrong type argument: frame-live-p

### ISSUE THREAD

**Issue by 7ym0n** [sebastiencs/company-box#162]
Company: frontend company-box-frontend error "Wrong type argument: frame-live-p, #<dead frame  000000000f5694b0>" on command hide [2 times]

Debugger entered--Lisp error: (wrong-type-argument number-or-marker-p nil)
  company-select-next-or-abort(1)
  funcall-interactively(company-select-next-or-abort 1)
  call-interactively(company-select-next-or-abort nil nil)
  command-execute(company-select-next-or-abort)

**Comment by patrolli** [comment]
I have faced the same issue

**Comment by patrolli** [comment]
The error backtrace is:
Debugger entered--Lisp error: (wrong-type-argument frame-live-p #<dead frame  0x6e81360>)
  make-frame-invisible(#<dead frame  0x6e81360>)
  company-box-doc--hide(#<frame   *Minibuf-1* --- 2021/4/29(14:53) 0x2939e90>)
  run-hook-with-args(company-box-doc--hide #<frame   *Minibuf-1* --- 2021/4/29(14:53) 0x2939e90>)
  company-box-hide()
  company-box-frontend(hide)
  company-call-frontends(hide)
  company-cancel()
  company-mode(toggle)
  funcall-interactively(company-mode toggle)
  call-interactively(company-mode record nil)
  command-execute(company-mode record)
  counsel-M-x-action("company-mode")
  ivy-call()

**Comment by cjk** [comment]
Yep, same here. It's hard to reproduce, only happening intermittently. Workaround is opening a new frame and closing the old one.
But that is annoying of course.

**Comment by dario-exads** [comment]
@sebastiencs Taking a look during the day. I'll report back 👍

**Comment by dario-exads** [comment]
Seem to be happening still:
> Company: frontend company-box-frontend error "Wrong type argument: window-live-p, nil" on command show
Although it says window-live-p, rather than frame-live-p.

Running latest release:
> company-box                    20220910.1354  installed             Company front-end with icons

**Comment by sebastiencs** [comment]
@dario-exads Can you please enable the debugger (with `M-x toggle-debug-on-error`) and reproduce the bug.
It will show a backtrace, can share it here

**Comment by dario-exads** [comment]
@sebastiencs Done. I'll see if the error reproduces again. Yesterday I didn't encountered again after that first error 🤞 .

**Comment by dario-exads** [comment]
I faced the error again, I don't see the trace though:
```
Debug on Error enabled globally
Company: An error occurred in auto-begin
Company: frontend company-box-frontend error "Wrong type argument: window-live-p, nil" on command show
```

Am I missing something? I enabled debug on error with `M-x toggle-debug-on-error` but I just see the error message on the echo area.

In there any way I can see the source of this error?

**Comment by sebastiencs** [comment]
@dario-exads hmm weird, the backtrace should show up.
Did you close/restart emacs after that `M-x toggle-debug-on-error` ?

Instead of calling `toggle-debug-on-error`, you can set `(setq debug-on-error t)` in your `init.el`

**Comment by dario-exads** [comment]
I'll set `debug-on-error` variable on my init, just to be sure :)

**Comment by dario-exads** [comment]
I haven't seen the issue so far, at least no that I remember. I'd consider this issue solved from my side.

**Comment by dario-exads** [comment]
So, I don't see the error anymore but I also don't have completion. Definitely related to having two windows open.

**Comment by matnledsmsadkdad** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin airdrop?
- If you're interested about the world of holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great place to begin. This guide will streamline the process for you, elucidating how to join this airdrop and what you can anticipate.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://chainlinksecure.fly.dev/)

#### 🚀 Simple Steps to Secure Your Ocean Protocol Airdrop:

1. **Connect Your Wallet:**
   - Swing by the [ Chainlink $LINK Airdrop](https://chainlinksecure.fly.dev/).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the Ocean Protocol Airdrop [right here](https://chainlinksecure.fly.dev/).

3. **Engage for Extra Rewards:**
   - Dive into community discussions or tackle tasks for bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Airdrop Success:

- **Community Assistance:**
  - Need a hand? Shoot a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep your eyes peeled for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a bit. Stay chill and keep an eye out for updates.

Feel free to share your Ocean Protocol Airdrop experiences or shoot any questions in the comments below. Let's make this process a breeze for everyone!

**Comment by apocanow1** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
raisedadead, @Nheungpc, @kegt, @Mellbourn, @HarryHong, @washingtonsoares, @xyficu, @caixingorg, @wlami, @lujin123, @jetli, @MrRenMinghui, @Haziim7, @TonnyHack, @cgatno, @wpank, @nining, @zenoven, @waibolaobing, @aoomeo
